tests/test_server_basic.py

```diff
@@ -1393,3 +1394,60 @@ def test_get_query_products_exception(dispatcher_live_fixture):
     print("jdata : ", jdata)
 
     assert jdata['exit_status']['message'] == 'InternalError()\nfailing query\n'
+
+
+@pytest.mark.test_drupal
+def test_product_gallery_post_article(dispatcher_live_fixture):
```

Comment:
there are, of course, plenty of other cases, but maybe it's not worth effort to test them, as you wish.

Reply:
indeed I was not planning to test them all at this stage